src/main/java/gov/nih/nci/evs/api/service/SparqlQueryManagerServiceImpl.java

```diff
@@ -2482,6 +2492,7 @@ public List<Concept> getAllConceptsWithoutCode(final Terminology terminology) th
     try {
       res = restUtils.runSPARQL(queryPrefix + query, getQueryURL());
     } catch (final Exception e) {
+      auditError(e, "getAllConceptsWithoutCode");
       e.printStackTrace();
```

Comment:
This is one of those situations - just remove the try complete.  it's not doing anything.  The error will bubble up.

Reply:
Others too are similar situations.  just look for e.printStackTrace().  if that's all that is happening, we don't need to try/catch